addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -15,7 +15,7 @@
                         threadLocalId="chatWindow.thread.localId"
                     />
                 </t>
-                <div class="o_ChatWindowHeader_item o_ChatWindowHeader_name" t-att-title="chatWindow.name">
+                <div class="o_ChatWindowHeader_item o_ChatWindowHeader_name" t-att-title="chatWindow.name" t-att-class="{'o_Clickable_ChatWindowHeaderName': chatWindow.thread and chatWindow.thread.channel_type === 'chat'}" t-on-click.stop="_onClickChatHeaderName">
```

Comment:
Don't redefine a conditional class `o_Clickable_ChatWindowHeaderName` with almost the same name as `o_ChatWindowHeader_name`. Define a modifier with appropriate name instead, such as `o-clickable`.

Condition should probably be on `thread.correspondent` here as well. There could exist `chat` with multiple members which would break this current condition.

Reply:
Also don't ever use stop propagation (notably this breaks global handlers that aren't capture). See comment about `markEventHandled` and `isEventHandled` for an alternative.